doc/rst/source/cookbook/coordinate-transformations.rst

```diff
@@ -1,297 +1,330 @@
+in GMT, list which parameters define them, and demonstrate how they are used to create simple plot axes. We will mostly
+be using :doc:`/basemap` (and occasionally :doc:`/plot`) to demonstrate the various transformations. Our illustrations
+may differ from those you reproduce with the same commands because of different settings in our ``gmt.conf`` file.
+Finally, note that while we will specify dimensions in inches (by appending **i**), you may want to use cm (**c**), or
+points (**p**) as :ref:`unit <cookbook/features:Dimension units>` instead.
 
 Cartesian coordinate transformations
 --------------------------------------------------------------------------------
 
 GMT Cartesian coordinate transformations come in three flavors:
 
--  Linear coordinate transformation
+-  :ref:`cookbook/coordinate-transformations:Linear coordinate transformation`
 
--  Log\ :math:`_{10}` coordinate transformation
+-  :ref:`cookbook/coordinate-transformations:Logarithmic coordinate transformation`
 
--  Power (exponential) coordinate transformation
+-  :ref:`cookbook/coordinate-transformations:Power (exponential) coordinate transformation`
 
-These transformations convert input coordinates *(x,y)* to
-locations *(x', y')* on a plot. There is no coupling between
-*x* and *y* (i.e., *x' = f(x)* and *y' = f(y)*);
-it is a **one-dimensional** projection. Hence, we may use separate
-transformations for the *x*- and *y*-axes (and
-*z*-axes for 3-D plots). Below, we will use the expression
-*u' = f(u)*, where *u* is either *x* or *y* (or
-*z* for 3-D plots). The coefficients in *f(u)* depend on the
-desired plot size (or scale), the chosen *(x,y)* domain, and the
-nature of *f* itself.
+These transformations convert input coordinates :math:`(x,y)` to locations :math:`(x', y')` on a plot. There is no
+coupling between :math:`x` and :math:`y` (i.e., :math:`x' = f(x)` and :math:`y' = f(y)`); it is a **one-dimensional**
+projection. Hence, we may use separate transformations for the :math:`x`- and :math:`y`-axes (and :math:`z`-axes for 3-D
+plots). Below, we will use the expression :math:`u' = f(u)`, where :math:`u` is either :math:`x` or :math:`y` (or
+:math:`z` for 3-D plots). The coefficients in :math:`f(u)` depend on the desired plot size (or scale), the chosen
+:math:`(x,y)` domain, and the nature of :math:`f` itself.
 
-Two subsets of linear will be discussed separately; these are a polar
-(cylindrical) projection and a linear projection applied to geographic
-coordinates (with a 360° periodicity in the *x*-coordinate). We
-will show examples of all of these projections using dummy data sets
-created with :doc:`/gmtmath`, a "Reverse
-Polish Notation" (RPN) calculator that operates on or creates table data:
+Two subsets of linear will be discussed separately; these are a polar (cylindrical) projection and a linear projection
+applied to geographic coordinates (with a 360° periodicity in the *x*-coordinate). We will show examples of all of these
```

Comment:
@PaulWessel, I couldn't find the linear polar (cylindrical) projection discussion. Can you provide some guidance as to where this is in the cookbook?

Reply:
All the non-geographic stuff is in the Chapter called 5. GMT Coordinate Transformations